pkg/controller/build/buildrequest/assets/Containerfile.on-cluster-build-template

```diff
@@ -29,11 +29,15 @@ metadata_expire=1m\n\
 baseurl=/tmp/mco-extensions/os-extensions-content/usr/share/rpm-ostree/extensions/\n\
 gpgcheck=0\n\
 skip_if_unavailable=False" > /etc/yum.repos.d/coreos-extensions.repo && \
-    chmod 644 /etc/yum.repos.d/coreos-extensions.repo && \
-    extensions="{{- range $index, $item := .ExtensionsPackages }}{{- if $index }} {{ end }}{{$item}}{{- end }}" && \
-    echo "Installing extension packages: $extensions" && \
-    rpm-ostree install $extensions && \
-    rm /etc/yum.repos.d/coreos-extensions.repo
+chmod 644 /etc/yum.repos.d/coreos-extensions.repo{{if .ExtensionsPackages}} && \    
+extensions="{{- range $index, $item := .ExtensionsPackages }}{{- if $index }} {{ end }}{{$item}}{{- end }}" && \
+echo "Installing extension packages: $extensions" && \
+rpm-ostree install $extensions{{end}}{{if and .KernelDefaultPackages .KernelPackages}} && \
+kernelDefaultPkgs="{{- range $index, $item := .KernelDefaultPackages }}{{- if $index }} {{ end }}{{$item}}{{- end }}" && \
+installKernelPkgs="{{- range $index, $item := .KernelPackages }}{{- if $index }} {{ end }}{{$item}}{{- end }}" && \	
+rpm-ostree install $installKernelPkgs && \
```

Comment:
I would replace this for
rpm-ostree override remove $kernelDefaultPkgs --install $installKernelPkgs 

the trick is that each package needs to have --install like:
https://gitlab.com/fedora/bootc/examples/-/blob/main/kernel-rt-rpm-ostree/Containerfile?ref_type=heads#L16

Another thing you can do is just use dnf install -y if we are sure on the versions this will be supported we will have dnf and the configurations for ```installonlypkgs=''
protect_running_kernel=False```